pkg/solana/txm/pendingtx_test.go

```diff
@@ -473,9 +473,6 @@ func TestPendingTxContext_on_finalized(t *testing.T) {
 		// Check it exists in finalized map
 		tx, exists := txs.finalizedErroredTxs[msg.id]
 		require.True(t, exists)
-		require.Len(t, tx.signatures, 2)
```

Comment:
removing because these checks were useless?

Reply:
Sort of. These fields weren't important to finalized/errored transactions so the checks were probably useless. But also removing because of the change to only store `finishedTx` now which just has id, retentionTs, and state.